sentry/values.yaml

```diff
@@ -1,4 +1,4 @@
-prefix:
+prefix: sentry-
```

Comment:
Sorry if it's not clear, but the `-` is already added automatically

Reply:
Got it :)